Model/Model.tex

```diff
@@ -858,17 +858,10 @@ \subsubsection{Other useful global variables}
 (Smalltalk at: #Smalltalk) == Smalltalk --> true
 \end{code}
 
+% ct: TODO DISCUSS: Shall we really recommend the usage of \ct{Sensor} any longer?}
```

Comment:
@codeZeilen I wonder for the other globals listed here, `Undeclared`, `SystemOrganization`, and maybe even `Transcript`, whether we should recommend their usage at all? There are `Smalltalk undeclared`, `SystemOrganizer default`, and `Project current transcript` that possibly are more idiomatic ways. Can we drop these descriptions, too? And if yes, should we explain the relevant concepts somewhere else?

Reply:
We definitely want to describe them somewhere else, including World and ActiveHand, they are all useful. The whole section has to be re-written to introduce the notion of global variables, and then explain why they are problematic, and then to introduce the idiomatic ways for all the objects in the old list.

Reply:
Later in this section, there is already a paragraph discouraging the use of global variables, but it's probably not precise enough, see <https://github.com/codeZeilen/SqueakByExample-english/blob/b8470833881043eafd33f06de62b51fee38e1313/Model/Model.tex#L880>.

World, ActiveHand, and Sensor are all Morphic-specific. Would the new description of `#currentWorld` and `#currentHand` maybe belong there?